### ISSUE TITLE
Ability to be alerted for a particular domain

### ISSUE THREAD

**Issue by robinwhittleton** [webcompat/webcompat.com#3224]
I’m currently in a position where I can either get fixes in or at least alert the appropriate team at ikea.com, so I’d love to set up an alert if new issues are filed. Is this functionality that could be considered?

**Comment by karlcow** [comment]
@robinwhittleton which type of alert would you like to have?

in no particular order.

* mail?
* notification?
* atom feed?
* uri with a json ready to consume by your own client?
* something else?


asking because it has been a recurrent discussion and we need understanding what would be the most practical.

**Comment by karlcow** [comment]
Poor man tool in the meantime.
https://github.com/webcompat/web-bugs/issues?q=is%3Aissue+ikea+in%3Atitle

**Comment by robinwhittleton** [comment]
Yeah, I’ve been visiting that search page when I remember (so every couple of months basically), but obviously I don’t want compat bugs to rely on one person thinking to check.

Mail would be the obvious candidate (and I could tie a team’s mailing list to it too) but a Slack bot would be nice. Although maybe I can just wrangle an existing Github integration to alert for new issues that match a filter? Hmm…

**Comment by karlcow** [comment]
This is also related to #132
Maybe more doable now given the webcompat-kibana instance.
